### DISCUSSION TITLE
Environment solve failed when installing MOOSE

### DISCUSSION THREAD

**Discussion by ver2king** [idaholab/moose#28823] [Q&A Getting Started]
### Bug Description

When installing MOOSE to the Linux distribution (as first image), creating the MOOSE environment failed and had to try with next repodata source. After this try, the solving environment just did not finish (as second image). 

![image](https://github.com/user-attachments/assets/0349dec1-a2c3-40eb-b5cc-b8d8054b0d3f)

![image](https://github.com/user-attachments/assets/4e2177f2-6f9e-4fe9-a954-3ff6aa2bec2c)

### Steps to Reproduce

_Steps are followed from this tutorial:_

https://mooseframework.inl.gov/getting_started/installation/conda.html

_Until the following command:_ 

conda create -n moose moose-dev=2024.10.01=mpich

### Impact

Prevention of installing MOOSE sucessfully

**Comment by milljm** [comment]
What version of Conda are you using?

```bash
conda info --all
```

Older versions of Conda, or Miniconda (Anaconda's variant) exhibit slow dependency solving. If you are using Anaconda, this can take anywhere up to 40 minutes.

**Comment by ver2king** [comment]
Please find the version of my Conda as below:

![image](https://github.com/user-attachments/assets/f521fa28-2508-4827-bad0-d6b0a618f5ec)

**Reply by milljm** [reply]
Yep, 23.x is old... I would advise updating to 24.x

What you want to see is the mamba solver:

```pre
$ conda info --all

     active environment : base
    active env location : /data/milljm/host_conda
            shell level : 1
       user config file : /home/milljm/.condarc
 populated config files : /data/milljm/host_conda/.condarc
                          /home/milljm/.condarc
          conda version : 24.9.1
    conda-build version : 24.7.1
         python version : 3.10.14.final.0
                 solver : libmamba (default)       <-------------- this
       virtual packages : __archspec=1=zen4
                          __conda=24.9.1=0
                          __cuda=12.4=0
                          __glibc=2.28=0
                          __linux=4.18.0=0
                          __unix=0=0
```
The version you have will eventually finish. But it can take upwards of 40 minutes to an hour.

**Reply by gsgall** [reply]
I had the same issue when trying to help a colleague update MOOSE. This worked for me.
